pkg/iam/oidc/api.go

```diff
@@ -132,7 +132,11 @@ func (m *OpenIDConnectManager) getIssuerCAThumbprint() error {
 	if err != nil {
 		return errors.Wrap(err, "connecting to issuer OIDC")
 	}
-
+	defer func() {
+		if response != nil {
```

Comment:
just curious, can it be that `response` is `nil` _and_ `err` is `nil`?

Reply:
you are right, that scenario is not happened. Otherwise we will have nil pointer in next line already `if response.TLS != nil`

Let me remove this nil check :)